### PR TITLE
Updates to tutorial demos for `beta3`

### PR DESCRIPTION
**Pull request by agubler**
Contains updates to dependencies that have been released as `next` after `beta2` in preparation for `beta3`.

Needs some tweaks to the text for the tutorials with code updates.

Mainly just changes to for `WidgetRegistry` -> `Registry` and some small changes for the injector tutorial.

The deployment of the changes: https://dist-fgwbwymlip.now.sh/

### CODE REVIEW

File: site/source/tutorials/1010_containers_and_injecting_state/index.md [hunk 1]
```diff
@@ -28,23 +28,27 @@ To begin our discussion, let's review the initial version of `App`:
```
Comment (agubler) [inline site/source/tutorials/1010_containers_and_injecting_state/index.md]: @dylans the changes to this tutorial could do with a real review as quite a lot has changed.

File: site/source/tutorials/1010_containers_and_injecting_state/index.md [hunk 2]
```diff
@@ -28,15 +28,19 @@ To begin our discussion, let's review the initial version of `App`:
...
 
 {% instruction 'Add the following to `ApplicationContext.ts`.' %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/ApplicationContext.ts' %}
 
 The code begins by importing some modules, including the `WorkerProperties` and `WorkerFormData` interfaces defined in the `Worker` and `WorkerForm` modules. These two interfaces define the shape of state that the `ApplicationContext` manages.
 
-The `ApplicationContext` class contains the state information. `ApplicationContext` has two private fields, `_workerData` and `_formData`, which contain the state, and two accessor methods to retrieve these fields. Finally, notice the two public methods that `ApplicationContext` exposes:
+The `ApplicationContext` contains the application state information. The base `Injector` class exposes a public method `get` that is used to determine what will be injected, by default the `get` method returns the value that is passed on `Injector` construction, i.e. `new Injector(myValueToInject);`. However, as the `ApplicationContext`, is itself the value the needs to be injected, the `get` method can be overridden to return the instance of `ApplicationContext`.
```
Comment (dylans) [inline site/source/tutorials/1010_containers_and_injecting_state/index.md]: `ApplicationContext`, is -> `ApplicationContext` is

File: site/source/tutorials/1010_containers_and_injecting_state/index.md [hunk 3]
```diff
@@ -108,11 +112,7 @@ Next, we need to address the fact that the container has two places to get prope
...
+The `getProperties` function receives two parameters. The first is the `payload` of the `injector` instance returned by the `injector`s `get()` method. For the demo application this is the instance of the `ApplicationContext` returned when we override the base `Injector` classes default `get()` method to return itself. The second is the `properties` that have been passed to the container via the normal mechanism, `w(Container, properties)`. The properties will implement the properties interface defined by the wrapped widget (for example `WorkerContainerProperties`). The `getProperties` function must then return an object that holds the properties that will be passed to the widget itself. In this example, we are ignoring the properties provided by the parent and returning the `workerData` stored by the `ApplicationContext`. More advanced use cases where both sources are used to generate the properties are also possible.
```
Comment (dylans) [inline site/source/tutorials/1010_containers_and_injecting_state/index.md]: For the demo application this is the instance of the `ApplicationContext` returned when we override the base `Injector` classes default `get()` method to return itself. -> For the demo application, this is the instance of the `ApplicationContext` returned when we override the base `Injector` class' default `get()` method to return itself.

File: site/source/tutorials/1020_registries/index.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ The first step is to create a `registry` that will be made available to the appl
 {% include_codefile 'demo/finished/biz-e-corp/src/main.ts' line:32 %}
 
 {% aside 'Registries Everywhere!' %}
-A `registry` can also be used to define widgets with non-visual responsibilities, such as state injection and routing. To learn more, take a look at the [container tutorial](../1010_containers_and_injecting_state/) and [routing tutorial](../comingsoon.html) .
+A `registry` can also be used to define injector that can be used to provide context for responsibilities such as state injection and routing. To learn more, take a look at the [container tutorial](../1010_containers_and_injecting_state/) and [routing tutorial](../comingsoon.html) .
```
Comment (dylans) [inline site/source/tutorials/1020_registries/index.md]: define injector -> define an injector

Comment (dylans) [reply site/source/tutorials/1020_registries/index.md]: that can be used to provide context for -> to provide context for

**Comment by agubler** [comment]
@dylans updated based on the review

**Pull request merged**